web/templates/gear_menu_popover.hbs

```diff
@@ -0,0 +1,174 @@
+<ul class="navbar-dropdown-menu" id="gear-menu-dropdown" aria-labelledby="settings-dropdown" data-simplebar>
+    <div class="org-info">
```

Comment:
So this should probably be a `<li class="org-info">` element, with the `<li>` elements here wrapped in probably an anonymous `<ul>` element.

Reply:
Yeah, changed.
```html
    <li class="org-info">
        <ul>
            <li class="org-name">{{realm_name}}</li>
...
```